docs/answer.md

```diff
@@ -0,0 +1,441 @@
+* ids: Id of the answer. e.g. 8940
+* filter: Filter you have chosen to filter fields in the response. e.g. default
+* preview: Preview parameter is used for development and testing purposes. When preview is set to true,
+           request will only simulate whether it is a valid request or not and will not change
+           anything on the site.
+
+**Sample request**
+```json
+{
+  "id": 55108393,
+  "preview": true
+}
+```
+
+**Sample response**
+```json
+{
+  "items": [
+    {
+      "owner": {
+        "reputation": 1294,
+        "user_id": 425465,
+        "user_type": "registered",
+        "profile_image": "https://www.gravatar.com/avatar/177b8d33a63edd4ff1c150130f554f83?s=128&d=identicon&r=PG",
+        "display_name": "harlam357",
+        "link": "https://stackoverflow.com/users/425465/harlam357"
+      },
+      "is_accepted": false,
+      "score": 1,
+      "last_activity_date": 1552329779,
+      "creation_date": 1552329779,
+      "answer_id": 55108393,
+      "question_id": 55105701
+    }
+  ],
+  "has_more": false,
+  "quota_max": 10000,
+  "quota_remaining": 9688
+}
+```
+
+Following is a sample REST request that can be handled by the downvoteAnswerById operation.
+
+**Related StackExchange API documentation**
+[https://api.stackexchange.com/docs/downvote-answer](https://api.stackexchange.com/docs/downvote-answer)
+
+### Accepting an answer
+Accept an existing answer.
+
+**acceptAnswerById**
+```xml
+<stackexchange.acceptAnswerById>
+    <id>{$ctx:id}</id>
+    <preview>{$ctx:preview}</preview>
+    <filter>{$ctx:filter}</filter>
+</stackexchange.acceptAnswerById>
+```
+
+**Properties**
+* ids: Id of the answer. e.g. 8940
+* filter: Filter you have chosen to filter fields in the response. e.g. default
+* preview: Preview parameter is used for development and testing purposes. When preview is set to true,
+           request will only simulate whether it is a valid request or not and will not change
+           anything on the site.
+
+**Sample request**
+```json
+{
+  "id": 55108393,
+  "preview": true
+}
+```
+
+**Sample response**
+```json
+{
+  "items": [
+    {
+      "owner": {
+        "reputation": 1294,
+        "user_id": 425465,
+        "user_type": "registered",
+        "profile_image": "https://www.gravatar.com/avatar/177b8d33a63edd4ff1c150130f554f83?s=128&d=identicon&r=PG",
+        "display_name": "harlam357",
+        "link": "https://stackoverflow.com/users/425465/harlam357"
+      },
+      "is_accepted": false,
+      "score": 1,
+      "last_activity_date": 1552329779,
+      "creation_date": 1552329779,
+      "answer_id": 55108393,
+      "question_id": 55105701
+    }
+  ],
+  "has_more": false,
+  "quota_max": 10000,
+  "quota_remaining": 9688
+}
+```
+
+Following is a sample REST request that can be handled by the acceptAnswerById operation.
+
+**Related StackExchange API documentation**
+[https://api.stackexchange.com/docs/accept-answer](https://api.stackexchange.com/docs/accept-answer)
```

Comment:
Add Sample configuration for one operation per resource
https://github.com/wso2-extensions/esb-connector-googlespreadsheet/blob/master/docs/workWithSpreadsheet.md#sample-configuration

Reply:
Fix with dd183b6